source/mir/random/engine/package.d

```diff
@@ -199,6 +270,147 @@ version(mir_random_test) unittest
     static assert(is(EngineReturnType!Random == size_t));
 }
 
+static if (THREAD_LOCAL_STORAGE_AVAILABLE)
+{
+    /++
+    Thread-local instance of the default $(LREF Random) allocated and seeded independently
+    for each thread. Requires $(LINK2 https://en.wikipedia.org/wiki/Thread-local_storage, TLS).
+    +/
+    alias rndGen = threadLocal!Random;
```

Comment:
How about shorter name without upper case? Maybe `rne`?

Reply:
`rng` probably is frequently used in user code, but `rne` should not

Reply:
Renamed it.